packages/ts/src/components/donut/config.ts

```diff
@@ -62,4 +68,6 @@ export const DonutDefaultConfig: DonutConfigInterface<unknown> = {
   emptySegmentAngle: 0.5 * Math.PI / 180,
   showBackground: true,
   backgroundAngleRange: undefined,
+  centralLabelsOffsetX: 0,
```

Comment:
I think it'll better better to have the default values as `undefined`. This way you will be able to customize the behavior of the component depending on the presence of these values. If they are not present, you can have some default logic. If they are present, you will simply override the offset.

Reply:
Great idea! Will do